src/java/fr/paris/lutece/portal/business/user/attribute/AttributeText.java

```diff
@@ -286,7 +287,8 @@ public List<AdminUserField> getUserFieldsData( String [ ] strValues, AdminUser u
     {
         List<AdminUserField> listUserFields = new ArrayList<>( );
         AdminUserField userField = new AdminUserField( );
-        AttributeService.getInstance( ).setAttributeField( this );
+        AttributeService attributeService = CDI.current( ).select( AttributeService.class ).get( );
```

Comment:
Do not use static calls; inject the service as a property since this class is a CDI bean.
@Inject 
private AttributeService attributeService;;